src/main/resources/de/qabel/desktop/ui/about/about.css

```diff
@@ -1,21 +1,123 @@
-.button-about {
-    -fx-font-weight: bolder;
-    -fx-pref-width:220px;
+
+.btn-about {
+    -fx-border-color:none;
+    -fx-pref-width:500px;
+    -fx-pref-height:60px;
     -fx-border-width: 1px 1px 1px 1px;
-    -fx-text-alignment:left;
     -fx-background-repeat: no-repeat;
     -fx-background-image: url('/img/chevron_right.png');
     -fx-background-position:right;
-    -fx-font-size: 14px;
+    -fx-alignment: TOP_LEFT;
+    -fx-padding: 15px 100px 5px 10px;
+}
+
+.hyperlink-box-about {
+    -fx-alignment: BOTTOM_LEFT;
+    -fx-border-color: transparent;
+    -fx-padding: 0;
+    -fx-underline: false;
+}
+
+.hyperlink-box-about .text {
+    -fx-fill: #222222;
+    -fx-font-size: 11;
+}
+
+.hyperlink-box-about:visited .text {
+    -fx-fill: #222222;
+}
+
+.hyperlink-box-about:hover .text {
+    -fx-fill: #222222;
+}
+
+.hyperlink-box-about:pressed .text {
+    -fx-fill: #222222;
 }
 
 .linkcontainer-about{
     -fx-alignment: center;
-    -fx-pref-width:586px;
-    -fx-pref-height:529px;
+    -fx-padding: 10px 10px 10px 10px;
+    -fx-min-width: 586px;
+    -fx-min-height: 529px;
+
+
 }
 
 .img-logo-about{
     -fx-fit-height:50px;
     -fx-fit-width:200px;
 }
+
+.scrollPaneAbout {
+
+}
+
+.label-about-top-left {
+    -fx-alignment: TOP_LEFT;
+}
+
+.label-about-bottom-left {
+    -fx-alignment: BOTTOM_LEFT;
+}
+
+.label-about-top-left  .text {
+    -fx-fill: #222222;
+    -fx-font-size: 11;
+}
+
+.label-about-bottom-left .text {
```

Comment:
you should better extract multiple classes here. like `.label-about` that has size and color defined and then classes like `top-left` to controls the alignment and boldness.

If you can`t create completely generic classes, it would be even better to create semantically abstract names and hierarchies like
`.box-button-about .license-title` instead of `.label-about-top-left` and
`.box-buttom-about .libs` instead of `.label-about-bottom-left`. This way, one could change the layout via CSS and not end up with definitions like `.label-about-top-left { -fx-alignment: TOP_RIGHT; }` for example. Defining a class `.label-about-bold` that just makes the text bold is just like setting the `style="-fx-font-weight: bold"` directly in the `fxml` document, so not preferrable.